app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```diff
@@ -1,330 +1,2 @@
 @import "mixins/prefixed-transform";
-
```

Comment:
I think this is probably fine... but just going to add that some `webkit-` prefixes are missing (that are present in the design system). e.g.
This is not specific to the Accordion - it's the same for all components (only observed here)

```
.js-enabled .govuk-accordion-nav__chevron {
    -webkit-box-sizing: border-box;
    box-sizing: border-box;
    ....
```
and
```
.js-enabled .govuk-accordion__section-header:hover {
    border-top-color: #b1b4b6;
    -webkit-box-shadow: inset 0 3px 0 0 #1d70b8;
    box-shadow: inset 0 3px 0 0 #1d70b8;
}
```

Reply:
Hmm it's a good spot, I would expect the same prefixes to pull through. I'm wondering if this is automated around support and since GOVUK supports 2 back `-webkit-` is decided it's not needed.